File: CodeInterviewApp/Controllers/MonacoContent.cs

using System.Collections.Generic;

namespace dotnetcore.Controllers
{
    public class MonacoContent
    {
        public int id;
        public string name;
        public string content;
        public string password;
        public int num_viewers;
        public int num_editors;
        public int current_version;
        public int line_number;
        public List<User> users = new List<User>();
    }
}

File: CodeInterviewApp/wwwroot/models/MonacoContent.ts
import { User } from "./User";

export class MonacoContent {
    id: number;
    name: string;
    content: string;
    password: string;
    num_viewers: number;
    num_editors: number;
    current_version:number;
    line_number:number;
    users:User[];
}